doc/builders/images/portableservice.section.md

```diff
@@ -0,0 +1,64 @@
+# pkgs.portableService {#sec-pkgs-portableService}
+
+`pkgs.portableService` is a function to create **"portable service images"**,
+as read-only, immutable, `squashfs` archives.
+
+systemd supports a concept of ["Portable Services"](https://systemd.io/PORTABLE_SERVICES/).
+"Portable Services" are a delivery method for system services that uses two specific features of container management:
+
+* Applications are bundled. I.e. multiple services, their binaries and
+  all their dependencies are packaged in an image, and are run directly from it.
+* Stricter default security policies, i.e. sandboxing of applications.
+
+This allows to use the benefits of nix, to build images for use on many recent Linux distributions.
+
+The primary tool for interacting with Portable Services is `portablectl`,
+and they are managed by the `systemd-portabled` system service.
+
+> **_NOTE:_** Portable services were introduced and only supported since systemd 239 (released on 2018-06-22).
+
+A very simple example of using `portableService` is described below:
+
+[]{#ex-pkgs-portableService}
+
+```nix
+pkgs.portableService {
+  pname = "demo";
+  version = "1.0";
+  units = [ demo-service demo-socket ];
+}
+```
+
+The above example will build an squashfs archive image in `result/$pname_$version.raw`. The image will contain the
+file-system structure as required by the portable service specification, and a sub-set of the nix store with all the
+dependencies of the two derivations in the `units` list.
+`units` must be a list of derivations, and their names must be prefixed with the service name ("demo" in this case),
+otherwise `systemd-portabled` will ignore them.
+
+> **_NOTE:_** The `.raw` file extension of the image is required by the portable services specification.
+
+Some other options available are:
+- `description`, `homepage`: are added to the `/etc/os-release` in the image and are shown by the portable services tooling.
```

Comment:
Please make the descriptions a paragraph under each list item and capitalize it. Otherwise it will display very densely. You can omit the colon then, it doesn't add information.

Reply:
is this what you had in mind?